### PR TITLE
updated verify signature function to handle unicode characters properly

### PR DESCRIPTION
**Pull request by cachrisman**

### CODE REVIEW

File: plivo.py [hunk 1]
```diff
@@ -20,15 +22,28 @@ class PlivoError(Exception):
     pass
 
 
-def validate_signature(uri, post_params, signature, auth_token):
-    for k, v in sorted(post_params.items()):
+def validate_signature(uri, params, signature, auth_token):
+    qs = urlparse(uri).query
+    if qs:
+        params_from_qs = dict(parse_qsl(qs))
```
Comment (wojcikstefan) [inline plivo.py]: https://www.plivo.com/docs/xml/request/#validation doesn't mention anything about combining query string params with form data. Is that really how it's supposed to be done? And if you *really* need to consider query string params in the signature, what happens if I e.g. set my Message URL to `https://example.com/message?somekey=someval`? Does Plivo ever append their own query string params to a URL?

File: plivo.py [hunk 2]
```diff
@@ -20,15 +22,28 @@ class PlivoError(Exception):
...
+    for k, v in sorted(params.items()):
+        k = k.encode('utf-8')
         uri += k + v
+        if v is None:
+            x = ''
+        elif isinstance(v, basestring):
+            x = v.encode('utf-8')
+        else:
+            x = str(v)
+        params[k] = x
```
Comment (wojcikstefan) [inline plivo.py]: Why would we override params? It's not expected that this function is going to alter any of the arguments we pass to it.

File: plivo.py [hunk 3]
```diff
@@ -20,15 +22,28 @@ class PlivoError(Exception):
     pass
 
 
-def validate_signature(uri, post_params, signature, auth_token):
-    for k, v in sorted(post_params.items()):
+def validate_signature(uri, params, signature, auth_token):
```
Comment (wojcikstefan) [inline plivo.py]: Could use a docstring (see #25 for an example).

File: plivo.py (hunk 1 above)
Comment (wojcikstefan) [inline plivo.py]: > Does Plivo ever append their own query string params to a URL?

I just realized that's what happens when you set your application's Message/Answer/Fallback/Hangup Method to GET

**Comment by kumaresan-plivo** [comment]
@wojcikstefan, This is being addressed by #26, so closing this PR

**Pull request closed without merging**